src/worker/ingestion/team-manager.ts

```diff
@@ -103,21 +103,33 @@ export class TeamManager {
         await this.cacheEventNamesAndProperties(team.id)
 
         if (!this.eventNamesCache.get(team.id)?.has(event)) {
+            const newId = new UUIDT().toString()
             await this.db.postgresQuery(
                 `INSERT INTO posthog_eventdefinition (id, name, volume_30_day, query_usage_30_day, team_id) VALUES ($1, $2, NULL, NULL, $3) ON CONFLICT DO NOTHING`,
-                [new UUIDT().toString(), event, team.id],
+                [newId, event, team.id],
                 'insertEventDefinition'
             )
+            await this.db.postgresQuery(
```

Comment:
I don't want to have to add definitions to an enterprise version table for non-enterprise users, but if they choose to become enterprise, they'll be missing these definitions if I don't send it along with the non-enterprise ones now, how should I go about this?